### DISCUSSION TITLE
[cli] Analyse all urls with a specific path

### DISCUSSION THREAD

**Discussion by ManAnRuck** [harlan-zw/unlighthouse#73] [Q&A] [answered]
Hi, is it possible to analyse all urls with a specific path?

`npx unlighthouse --site unlighthouse.dev/guide/getting-started --urls /guide`
get only one page: https://unlighthouse.dev/guide/getting-started/

Without `--urls` I get all pages

but I want all pages with the /guide path:
- https://unlighthouse.dev/guide/getting-started
- https://unlighthouse.dev/guide/getting-started/integrations
- https://unlighthouse.dev/guide/getting-started/how-it-works
- ...

something like: `npx unlighthouse --site unlighthouse.dev/guide/getting-started --urls /guide/*`

**Comment by harlan-zw** [comment, marked as answer]
Hey @ManAnRuck, thanks for the question.

You can use the `--include-urls` option which supports regex :) 

`npx unlighthouse@0.7.1 --site unlighthouse.dev/guide/getting-started --include-urls /guide/*`

Let me know if you have any issues with that

**Reply by ManAnRuck** [reply]
Thank you @harlan-zw.

It works with a small adjustment:
`npx unlighthouse@0.7.1 --site unlighthouse.dev/guide/getting-started --include-urls "/guide/*"`

**Reply by harlan-zw** [reply]
Oops, thanks :laughing:
